pkg/topology/full/full.go

```diff
@@ -149,6 +155,60 @@ func (d *driver) ClosestPeer(addr swarm.Address) (swarm.Address, error) {
 	return closest, nil
 }
 
+func (d *driver) Close() error {
```

Comment:
This function should be called by node.Bee.Shutdown and in pkg/topology/full/full_test.go tests.

Reply:
I did not call it on purpose, as I see we don't really have a consistent shuwdown strategy across the project. But I will add it anyway, and let's see how it fits in the future.

Reply:
actually the shutdown strategy is in `node.go`. we should obviously be mindful about inter-package dependencies when shutting down, but this is supposed to be handled by the shutdown sequence in the shutdown in `node.go`

Reply:
We have a shutdown strategy, and it should be consistent. Every service started by node.NewBee should be closed in node.Bee.Shutdown. If there is something that is missing we should address it as soon as possible.

Also, every test should cleanup resources that it acquires when it ends. If there are tests that do not do that, we should address them as soon as possible.

Reply:
Done!